src/MIDebugEngine/Engine.Impl/Variables.cs

```diff
@@ -469,6 +476,20 @@ internal async Task Eval(enum_EVALFLAGS dwFlags = 0)
                 }
                 else
                 {
+                    int numElements = 200;
+                    if (dwDAPFlags.HasFlag(DAPEvalFlags.CLIPBOARD_CONTEXT))
+                    {
+                        string showPrintElementsResult = await MIDebugCommandDispatcher.ExecuteCommand("show print elements", _debuggedProcess, ignoreFailures: true);
+                        // Possible values for 'numElementsStr'
+                        // "Limit on string chars or array elements to print is <number>."
+                        // "Limit on string chars or array elements to print is unlimited."
+                        string numElementsStr = Regex.Match(showPrintElementsResult, @"\d+").Value;
+                        if (int.TryParse(numElementsStr, out numElements))
```

Comment:
>) [](start = 73, length = 1)

`&& numElements != 0`

Reply:
If you can't parse it, then what is the behavior that happens?

Reply:
Then TryParse will fail and we will just default to the current print element value.